examples/xpath-selectors/xpath-selector.js

```diff
@@ -0,0 +1,19 @@
+import { Selector } from 'testcafe';
+
+const elementByXPath = Selector(xpath => {
+    const iterator = document.evaluate(xpath, document, null, XPathResult.UNORDERED_NODE_ITERATOR_TYPE, null )
+    const items = [];
+
+    let item = iterator.iterateNext();
+
+    while (item) {
+        items.push(item);
+        item = iterator.iterateNext();
+    }
+
+    return items;
+});
+
+export default function (xpath) {
+    return Selector(elementByXPath(xpath));
```

Comment:
@miherlosev So do we have to wrap `elementByXPath` in `Selector` once again?

Reply:
Yes. It allows us to apply the Selector methods to the call result.
```js
XPathSelector('//input[@type="checkbox"]').nth(1); // It's not possible to call '.nth(1) for the nodes collection.
```